app/src/main/java/com/github/se/polyfit/ui/navigation/Route.kt

```diff
@@ -3,9 +3,15 @@ package com.github.se.polyfit.ui.navigation
 object Route {
   const val Home = "Home"
   const val Register = "Register"
+
+  const val Overview = "Overview"
+  const val Map = "Map"
```

Comment:
should use a different name from Map, since it is a data type from Kotlin we don't want to overwrite